packages/authorization-platformstate-writer/src/consumerServiceV1.ts

```diff
@@ -110,6 +110,7 @@ export async function handleMessageV1(
           clientPurposesIds,
           version: msg.version,
           updatedAt: new Date().toISOString(),
+          adminId: clientEntry.adminId,
```

Comment:
Assumption  `adminId` is always optional. 
V1 messages should be deprecated. Do we really want to update them with adminId? What use case do you want to handle? Are V1 messages still used in any way?